flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaSinkBuilderTest.java

```diff
@@ -73,9 +72,9 @@ public void testPropertyHandling() {
                         .setKafkaProducerConfig(testConf)
                         .setProperty("k2", "correct"),
                 p -> {
-                    Arrays.stream(DEFAULT_KEYS).forEach(k -> assertTrue(k, p.containsKey(k)));
-                    assertEquals("v1", p.get("k1"));
-                    assertEquals("correct", p.get("k2"));
+                    Arrays.stream(DEFAULT_KEYS).forEach(k -> assertThat(p).containsKey(k));
+                    assertThat(p.get("k1")).isEqualTo("v1");
+                    assertThat(p.get("k2")).isEqualTo("correct");
```

Comment:
Here `containsEntry` could be used + joined
```suggestion
                    assertThat(p).containsEntry("k1", "v1").containsEntry("k2", "correct");
```